examples/pubsub/gcp-pubsub/main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"github.com/NYTimes/gizmo/examples/nyt"
+	"github.com/NYTimes/gizmo/pubsub"
+	"github.com/sirupsen/logrus"
+	"github.com/NYTimes/gizmo/pubsub/gcp"
+	"context"
+	"fmt"
+)
+
+const (
+	projID = "my-project"
+	topic = "my-topic"
+	subscription = "my-subscription"
```

Comment:
Most users won't hard code some of this information. Would you mind adding a few lines in the example to highlight the use of https://godoc.org/github.com/NYTimes/gizmo/pubsub/gcp#Config and/or https://godoc.org/github.com/NYTimes/gizmo/pubsub/gcp#LoadConfigFromEnv?